test/tests/functional/pbs_hook_management.py

```diff
@@ -48,6 +48,17 @@
 from tests.functional import JOB
 
 
+def get_hook_body(hook_msg):
+    hook_body = """
+    import pbs
+    e = pbs.event()
+    m = e.management
+    pbs.logmsg(pbs.LOG_DEBUG, '%s')
+    """ % hook_msg
+    hook_body = textwrap.dedent(hook_body)
+    return hook_msg
```

Comment:
This should be hook_body instead

Reply:
Interesting that that got through.  I am trying to find a way to make sure my changes get correctly propagated to my docker container.  The test is now failing.  For now I will just rebuild the entire thing, which doesn't take long at all.

Reply:
This should be fixed now.  There were other problems in the test runner that I just fixed while correcting this.